### ISSUE TITLE
Usage of the 'filterlabels' option.

### ISSUE THREAD

**Issue by ryanpxx** [AKSarav/Kube-Node-Usage#19]
Hello, 

I would like to "filter by label". Sadly the usage or syntax is unclear to me. Can you perhaps provide a few examples?

I can't filter on standard EKS node labels like e.g. `failure-domain.beta.kubernetes.io/zone=eu-central-1b` or similar. 

For example, this does not work:

``` 
KubeNodeUsage -filterlabels="failure-domain.beta.kubernetes.io/zone=eu-central-1b"
``` 
Working with: Version: 3.0.1

Thanks, 

Phil.

**Comment by ryanpxx** [comment]
OK, just noticed this: https://github.com/AKSarav/Kube-Node-Usage/pull/15

**Comment by AKSarav** [comment]
I am working on the filter labels right now - Please check out the new Label as a Column feature added on v3.0.2

**Comment by AKSarav** [comment]
This is released with v3.0.2 - closing the issue
